builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -578,11 +580,18 @@ function store.load()
 	store.aliases = {}
 
 	for _, package in pairs(store.packages_full) do
+		package.url = base_url .. "/packages/" ..
+			package.author:lower():encode_uri_component() ..
+			"/" .. package.name ..
+			"/releases/" .. package.release .. "/download/"
+
 		local name_len = #package.name
 		if package.type == "game" and name_len > 5 and package.name:sub(name_len - 4) == "_game" then
-			package.id = package.author:lower() .. "/" .. package.name:sub(1, name_len - 5)
+			package.id = package.author:lower():encode_uri_component() ..
```

Comment:
id is used to track packages locally, and should never be used in a URL as case info is lost